### PR TITLE
Log service version

### PR DESCRIPTION
**Pull request by pawbu**

### CODE REVIEW

File: job_executor/config/log.py [hunk 1]
```diff
@@ -30,7 +23,7 @@ def format(self, record: logging.LogRecord) -> str:
                     record.created,
                     tz=datetime.timezone.utc,
                 ).strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3]
-                + "Z",
+                              + "Z",
```
Comment (DanielElisenberg) [inline job_executor/config/log.py:26]: ruff did this? 🤔  feels like an intellij thing

**Review by DanielElisenberg — APPROVED** [review]
Small formatting question, but nothing consequential, looks good :100:

File: job_executor/config/log.py (hunk 1 above)
Comment (pawbu) [reply job_executor/config/log.py:26]: Yes, it's Ruff plugin that misbehaves in Intellij I think, I'll fix 🏃

**Review by DanielElisenberg — APPROVED** [review]
:100:

**Pull request merged**